src/java.compiler/share/classes/javax/tools/StandardJavaFileManager.java

```diff
@@ -190,16 +190,16 @@
      * Returns file objects representing the given paths.
      *
      * @implSpec
-     * The default implementation converts each path to a file and calls
-     * {@link #getJavaFileObjectsFromFiles getJavaObjectsFromFiles}.
-     * IllegalArgumentException will be thrown if any of the paths
-     * cannot be converted to a file.
+     * The default implementation lazily converts each path to a file and calls
+     * {@link #getJavaFileObjectsFromFiles(Iterable) getJavaFileObjectsFromFiles}.
+     * {@linkplain IllegalArgumentException IllegalArgumentException} will be thrown
```

Comment:
Suggest using with `{@link}` or `{@code}` but not `{@linkplain}`.
Given that there is a `@throws` which will link it, `{@code}` may be enough.